toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -114,18 +115,20 @@ else
 fi
 
 # ============ Populate SRPMS ============
-# Populate ${repo_path}/build/INTERMEDIATE_SRPMS with SRPMs, that can be used to build RPMs in the container
-pushd "${repo_path}/toolkit"
-echo "Populating Intermediate SRPMs..."
-if [[ ( ! -f "${repo_path}/toolkit/out/tools/srpmpacker" ) ]]; then build_tools; fi
-make input-srpms SRPM_FILE_SIGNATURE_HANDLING="update" > /dev/null
-popd
+# Populate ${repo_path}/build/INTERMEDIATE_SRPMS with SRPMs, that can be used to build RPMs in the container (only required in build mode)
+if [[ "${mode}" == "build" ]]; then
+    pushd "${repo_path}/toolkit"
+    echo "Populating Intermediate SRPMs..."
+    if [[ ( ! -f "${repo_path}/toolkit/out/tools/srpmpacker" ) ]]; then build_tools; fi
+    make input-srpms SRPM_FILE_SIGNATURE_HANDLING="update" > /dev/null
```

Comment:
I'd still like to see at least `SRPM_PACK_LIST` handled here. It is also very important for quickly working on a specific package (a slow internet connection can mean it takes ~1hr to finish building `input-srpms`).